dbms/src/main/java/org/polypheny/db/PolyphenyDb.java

```diff
@@ -346,6 +417,18 @@ public void join( final long millis ) throws InterruptedException {
                 throw new RuntimeException( "There was no catalog submitted, aborting." );
             }
 
+            if ( AutoDocker.getInstance().isAvailable() ) {
+                if ( testMode ) {
+                    resetDocker = true;
+                    Catalog.resetDocker = true;
+                }
+                boolean success = AutoDocker.getInstance().doAutoConnect();
+                if ( testMode && !success && !System.getenv( "RUNNER_OS" ).equals( "Windows" ) ) {
```

Comment:
Please add a comment to the code for the `!System.getenv( "RUNNER_OS" ).equals( "Windows" )` part.